src/services/upper-bar-manager/upper-bar-manager.tsx

```diff
@@ -6,30 +6,35 @@ import { DisplayedBar } from './ui/displayed-bar/displayed-bar.component';
 import { KalturaPluginNames } from '../../types/ui-managers-config';
 const { ReservedPresetAreas, ReservedPresetNames } = ui;
 
+const presets = Object.values(ReservedPresetNames).filter(
```

Comment:
const should be uppercase by convention and find more descriptive name for the filtered presets